src/components/AddToEmbeddedPluginButton/AddToEmbeddedPluginButton.vue

```diff
@@ -111,24 +114,39 @@ async function onConfirmedToAdd() {
   }
 
   if (elevatorCallbackType.value === "lti") {
-    const data = await api.postLtiPayload({
-      fileObjectId: props.fileHandlerId,
-      excerptId: "",
-      returnUrl: returnUrl.value ?? "",
-    });
-
-    ltiContentItems.value = JSON.stringify(data);
-
-    nextTick(() => {
-      if (!returnForm.value) {
-        addingToPluginStatus.value = "error";
-        throw new Error("Return form not set");
-      }
-
-      returnForm.value.submit();
-      addingToPluginStatus.value = "success";
-      toastStore.addToast({ message: "Added to Canvas" });
-    });
+
+
+    if (elevatorLTIVersion.value == "1.3") {
+      const data = await api.postLtiPayload13({
+        fileObjectId: props.fileHandlerId,
+        excerptId: "",
+        returnUrl: returnUrl.value ?? "",
+        launchId: elevatorLaunchId.value ?? "",
+        userId: userId.value ?? "",
+      });
+
+      document.body.innerHTML += data;
+      (document.getElementById('auto_submit') as HTMLFormElement)?.submit();
+    }
+    else {
+      const data = await api.postLtiPayload({
+        fileObjectId: props.fileHandlerId,
+        excerptId: "",
+        returnUrl: returnUrl.value ?? ""
+      });
+      ltiContentItems.value = JSON.stringify(data);
+
+      nextTick(() => {
+        if (!returnForm.value) {
+          addingToPluginStatus.value = "error";
+          throw new Error("Return form not set");
+        }
+
+        returnForm.value.submit();
+        addingToPluginStatus.value = "success";
+        toastStore.addToast({ message: "Added to Canvas" });
+      });
```

Comment:
Should we show a success message to the user in LTI1.3 as well? Or maybe this isn't required because they'll see something from the `document.body.innerHTML += data` line above?

Reply:
nevermind!